src/validation_workflow/api_request.py

```diff
@@ -18,9 +21,10 @@
 
 class ApiTest:
 
-    def __init__(self, request_url: str) -> None:
+    def __init__(self, request_url: str, version: str) -> None:
         self.request_url = request_url
-        self.apiHeaders_auth = {"Authorization": "Basic YWRtaW46YWRtaW4="}  # default user/pass "admin/admin" in Base64 format
+        self.password = base64.b64encode(f"admin:{get_password(version)}".encode("utf-8")).decode("utf-8")
+        self.apiHeaders_auth = {"Authorization": f'Basic {self.password}'}  # user/pass "admin/pass" in Base64 format fetched from get_password() method
```

Comment:
Used this logic without altering https://github.com/opensearch-project/opensearch-build/blob/main/src/test_workflow/integ_test/utils.py

Reply:
I am not saying you need to alter, I am saying you can add a second version in return of base64 version. Instead of writing a base64 conversion within validation workflow.